docs/web3.eth.account.rst

```diff
@@ -83,9 +83,9 @@ account you can just generate a random number that acts as a private key.
 
 To create a private key using web3.py and command line you can do:
 
-```shell
-python -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.toHex(acc.privateKey)}, account={acc.address}')"
-```
+.. code-block:: shell
+
+  python3 -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.to_hex(acc._private_key)}, account={acc.address}')"
```

Comment:
Thanks for this update! Please stick with `python` here to keep it generic and consistent.

Also noticed the return value a few lines down also needs help, perhaps just remove the ticks on that block entirely.

Reply:
yea formatting is messy, it needs more work for sure, unfortunately I don't know much about `.rst`, but there was also `toHex` and `privateKey` properties used, which doesn't exist on latest version. encountered it while trying it and thought I'd send a quick fix